frontend/public/actions/common.ts

```diff
@@ -0,0 +1,14 @@
+import { action } from 'typesafe-actions';
```

Comment:
FYI @spadgett this is to break dependency cycle between `public/actions/ui.ts` and `public/actions/features.ts` by introducing a common module containing the least amount of code to break the cycle.